### DISCUSSION TITLE
PS1, prevent games being listed twice

### DISCUSSION THREAD

**Discussion by EdgarZev** [OnionUI/Onion#558] [Ideas]
Heya, 

Love your work!

Can you allow PS games to only list once when you're using .bin and .cue files?
It seems to list the games twice when they are in the folder.

cheers

**Comment by Wapp92** [comment]
Your problem is with games that have multiple bin files, ie track 1 track 2 ect. 
I put those in there own folder, it prevent cluster. 
But i am with you, not listing the .bin files should solve the issue.
